src/libs/actions/PersonalDetails.js

```diff
@@ -66,10 +67,6 @@ function getDisplayName(login, personalDetail) {
         return userLogin;
     }
 
-    if (userDetails.displayName) {
-        return userDetails.displayName;
-    }
```

Comment:
Not sure if we need this I think this is a generated field anyway and this was preventing us from updating it when the firstName or lastName changes.